bot/lords_and_knights_v3.rb

```diff
@@ -96,44 +97,45 @@ def choose_tab(title)
 
     def choose_building(titles)
       titles = [titles] unless titles.is_a?(Array)
-      logger.debug(": choose_building #{titles.join(', ')}")
-      result = nil
-      available_buildings = all('#menu-section-general-container > .menu-section > .menu--content-section .menu-list-element.menu-list-element-basic.clickable.with-icon-left.with-icon-right')
+      
+      logger.debug "   * choose_building #{titles.join(', ')}"
+      available_buildings = all('.menu-list-element.menu-list-element-basic.clickable.with-icon-left.with-icon-right') 
```

Comment:
(optional)

@t3hExi as i remember I wanted to `#menu-section-general-container` like a namespace. It was possible to have multiple menu lists in same time.

```suggestion
      available_buildings = all('#menu-section-general-container .menu-list-element.menu-list-element-basic.clickable.with-icon-left.with-icon-right') 
```